### PR TITLE
ADD migration documentation for triconfig

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: https://github.com/storybookjs/storybook/issues/9030

## What I did
ADD migration documentation for tri-config

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -74,8 +75,73 @@
...
+```js
+module.exports = {
+  stories: ['../**/*.stories.js'],
+  presets: ['@storybook/addon-docs/preset'],
+};
+```
+
+#### Using preview.js
+
+If after migrating the imports/requires of your stories to `main.js` you're left with some code in `config.js` it's likely the usage of `useParameters` & `addDecorator`.
```
Comment (shilman) [inline MIGRATION.md]: addParameters?

File: MIGRATION.md [hunk 2]
```diff
@@ -74,8 +75,73 @@
     - [Packages renaming](#packages-renaming)
     - [Deprecated embedded addons](#deprecated-embedded-addons)
 
+
 ## From version 5.2.x to 5.3.x
 
+### To tri-config
```
Comment (shilman) [inline MIGRATION.md]: Tri-config configuration?

File: MIGRATION.md [hunk 3]
```diff
@@ -74,8 +75,73 @@
...
+
+These files are now soft-deprecated, (*they still work, but over time we will promote users to migrate*):
+
+- `config.js` has been renamed to `preview.js`.
+- `addons.js` has been renamed to `manager.js`.
+- `presets.js` has been renamed to `main.js`.
+
+#### Using main.js
+
+this is what a basic `main.js` looks like:
```
Comment (shilman) [inline MIGRATION.md]: Now `main.js` is now the main point of configuration for Storybook. This [...]

File: MIGRATION.md (hunk 1 above)
Comment (ndelangen) [reply MIGRATION.md]: ✅

File: MIGRATION.md (hunk 2 above)
Comment (ndelangen) [reply MIGRATION.md]: ✅

File: MIGRATION.md [hunk 4]
```diff
@@ -74,8 +75,73 @@
...
+
+If after migrating the imports/requires of your stories to `main.js` you're left with some code in `config.js` it's likely the usage of `useParameters` & `addDecorator`.
+
+This is fine, rename `config.js` to `preview.js`.
+
+This file can also be used to inject global stylesheets, fonts etc, into the preview bundle.
+
+#### Using manager.js
+
+If you are setting storybook options in `config.js`, especially `theme`, you should migrate it to `manager.js`:
```
Comment (shilman) [inline MIGRATION.md]: Add a sentence on why this is an improvement

File: MIGRATION.md (hunk 3 above)
Comment (ndelangen) [reply MIGRATION.md]: ✅

File: MIGRATION.md (hunk 4 above)
Comment (ndelangen) [reply MIGRATION.md]: ✅

**Comment by ndelangen** [comment]
@shilman good to merge?

File: MIGRATION.md [hunk 5]
```diff
@@ -74,8 +75,76 @@
...
+
+These files are now soft-deprecated, (*they still work, but over time we will promote users to migrate*):
+
+- `config.js` has been renamed to `preview.js`.
+- `addons.js` has been renamed to `manager.js`.
+- `presets.js` has been renamed to `main.js`.
+
+#### Using main.js
+
+Now main.js is now the main point of configuration for Storybook. This is what a basic `main.js` looks like:
```
Comment (daKmoR) [inline MIGRATION.md]:
```suggestion
`main.js` is now the main point of configuration for Storybook. This is what a basic `main.js` looks like:
```

**Review by shilman — APPROVED** [review]
LGTM!

**Pull request merged**